pyprof/nvtx/nvmarker.py

```diff
@@ -248,4 +249,15 @@ def init():
         if isfunc(cls, 'forward'):
             add_wrapper(cls, 'forward')
 
+    # Patch nvidia apex library only if imported
+    if 'apex' in sys.modules:
+        print("Patching apex")
+        patchClass(sys.modules['apex'])
+
+    # Patch nvidia dali library only if imported
+    for mod in sys.modules:
+        if 'nvidia.dali' in mod:
+            print(f'Patching {mod}')
+            patchClass(sys.modules[mod])
+
```

Comment:
None of this should go in until we have an example where it works.

Reply:
i agree. set title to WIP